core/system_metric/sys_metric_stat.go

```diff
@@ -49,6 +58,11 @@ var (
 	TotalMemorySize    = getTotalMemorySize()
 
 	ssStopChan = make(chan struct{})
+
+	isContainer             bool
+	preSysTotalCpuUsage     atomic.Value
+	preContainerCpuUsage    atomic.Value
```

Comment:
How about `atomic.Float64`?("go.uber.org/atomic")

Reply:
yeap,fixed it